game/CMakeLists.txt

```diff
@@ -182,6 +200,7 @@ if("AUTO" STREQUAL "${ENABLE_WEBSERVER}")
 		message(STATUS "Webserver support: Enabled (automatically found)")
 	else()
 		message(STATUS "Webserver support: Disabled (cpprestsdk not found)")
+		SET (ENABLE_WEBSERVER OFF FORCE)
```

Comment:
This shouldn't be necessary?

Reply:
I added it in an attempt to have performous not link against windows networking libraries when building without the webserver.